packages/light.js/src/rpc/other/makeContract.ts

```diff
@@ -98,7 +97,7 @@ const makeContractWithApi = memoizee(
               args
             ),
             ...options
```

Comment:
Not sure what's the best way to get rid of the tslint warning: *An argument for 'options' was not provided* (`options.passphrase`, which is required, but `options` here is defined as `const options = args.length === method.inputs.length + 1 ? args.pop() : {};`)